### PR TITLE
Bau add utils

### PR DESCRIPTION
**Pull request by srh-sloan**
Adding fsd-utils as a dependency and then implementing the following from it:
- Config class for loading app config across different envs
- db_fixtures for dynamic loading of test data and cleanup

### CODE REVIEW

File: config/__init__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# flake8: noqa
+from os import environ
+
+FLASK_ENV = environ.get("FLASK_ENV")
+print(FLASK_ENV)
+
+match FLASK_ENV:
+    case "tasks" | "development":
```
Comment (Adam-W1) [inline config/__init__.py:8]: is that meant to be "test"?

File: requirements.in [hunk 2]
```diff
@@ -54,3 +54,7 @@ Flask-Babel>=2.0.0
 #   Forms
 #-----------------------------------
 Flask-WTF==1.2.1
+#-----------------------------------
```
Comment (Adam-W1) [inline requirements.in:57]: minor but there are two forms heading here

File: config/__init__.py (hunk 1 above)
Comment (srh-sloan) [reply config/__init__.py:8]: No, it's for when we run stuff with invoke

**Review by Adam-W1 — APPROVED** [review]
LGTM :)

**Pull request merged**